src/components/ActionBar/ActionBar.jsx

```diff
@@ -92,7 +92,7 @@ class ActionsBar extends PureComponent {
             {isCameraEnabled ? <Video /> : <VideoOff />}
           </ActionBarListItem>
           <ActionBarListItem
-            onClick={endCall}
+            onClick={() => endCall(this.props.history)}
```

Comment:
please move this line into separate method (something like `handleEndCall`). It's because every time when `render` method will be called a new function will be created and stored in memory